warehouse/migrations/versions/34c3175f4bea_remove_oidc_association_on_project_.py

```diff
@@ -0,0 +1,51 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Remove OIDC Association on Project Delete
+
+Revision ID: 34c3175f4bea
+Revises: 1fdecaf73541
+Create Date: 2024-04-09 21:38:26.340992
+"""
+
+from alembic import op
+
+revision = "34c3175f4bea"
+down_revision = "1fdecaf73541"
+
+
+def upgrade():
+    op.drop_constraint(
+        "oidc_publisher_project_association_project_id_fkey",
+        "oidc_publisher_project_association",
+        type_="foreignkey",
+    )
+    op.create_foreign_key(
+        None,
```

Comment:
Just curious, did you change the constraint name to be `None` or was this auto-generated? I'm just surprised it's different than when it was first added.

Reply:
It's autogenerated, and I confirmed that the constraint name remains `oidc_publisher_project_association_project_id_fkey` after the migration completes.

Reply:
Weird! 🤷